### ISSUE TITLE
Modbutton errors not shown to user

### ISSUE THREAD

**Issue by Geo1088** [toolbox-team/reddit-moderator-toolbox#379]
When an action is attempted via the mod button and Reddit returns one or more errors in response, we currently just tell the user "This failed, do you want to retry?" even if the errors can't possibly be fixed by retry (e.g. user is a mod, message too long). We should come up with a UI for presenting the errors to the user directly (they're returned with nice strings that can be displayed!) and letting them fix the issue.

This is gonna be a bit weird with the current multi-sub nature of the modbutton window, but we should be able to figure something out.

**Comment by creesch** [comment]
Hasn't this been addressed (partially) by #383 ?

**Comment by Geo1088** [comment]
> Hasn't this been addressed (partially) by #383 ? 

Not really. #383 added client-side length checks before sending a request, but when requests fail for other reasons, we're still just displaying a generic message and ignoring the error info Reddit sends.

**Comment by creesch** [comment]
Gotcha, I should have checked the code for that one first. I just assumed it went of API errors.
